src/mixin.ts

```diff
@@ -73,14 +81,15 @@ export function mixin(Vue: VueConstructor) {
     const setup = vm.$options.setup!
     const ctx = createSetupContext(vm)
 
-    // mark props
-    markReactive(props)
+    // fake reactive for `toRefs(props)`
+    def(props, '__props_reactive__', true)
```

Comment:
Let's put `__props_reactive__` inside `src/utils/symbols.ts` instead of a magic string 😄